packages/zoe/src/contractFacet/exit.js

```diff
@@ -26,12 +26,21 @@ export const makeExitObj = (proposal, zoeSeatAdmin, zcfSeatAdmin) => {
 
   if (exitKind === 'afterDeadline') {
     // Automatically exit the seat after deadline.
-    E(proposal.exit.afterDeadline.timer).setWakeup(
-      proposal.exit.afterDeadline.deadline,
-      harden({
-        wake: exitFn,
-      }),
-    );
+    E(proposal.exit.afterDeadline.timer)
+      .setWakeup(
+        proposal.exit.afterDeadline.deadline,
+        harden({
+          wake: exitFn,
+        }),
+      )
+      .catch(err => {
+        console.error(
+          `The seat could not be made with the provided timer ${proposal.exit.afterDeadline.timer} and deadline ${proposal.exit.afterDeadline.deadline}`,
+        );
+        console.error(err);
+        exitFn();
```

Comment:
`exitFn` does an `exit`. Here, I think you want to do a `kickOut`.

Reply:
Ah, very good point! I will fix that